### PR TITLE
Add basic content to README

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Add basic content to README

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+```bash
+  ./gradlew build
+```
+
+### Running the application
+
+Create the image of the application by executing the following command:
+
+```bash
+  ./gradlew assemble
```
Comment (rkondratowicz) [inline README.md]: does this actually *run* the app?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
-# bulk-scan-orchestrator
+# Bulk Scan Orchestrator
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
+
+## Purpose
+
+The purpose of this application is to retrieve information about envelopes uploaded to Bulk Scanning
+(via Azure Service Bus queue) and manage the related CCD cases.
```
Comment (doncem) [inline README.md]: Isn't it `uploaded to Azure Blob Storage` or `uploaded to HMCTS Document Store via Bulk Scan Processor`?

Comment (nitinprabhu) [reply README.md]: Bulk scanning orchestrator does below 
1. Consume SB message and retrieve Envelope information using envelope id.
2. Based on envelope metadata create an exception record or updates an existing case in CCD.
3. Send envelope process event to bulk scanning processor.
4. Handle failed event(to be sent to bsp), failed envelope retrieval ,failed case/exception records.

**Comment by rkondratowicz** [comment]
what's going on with this PR, @lgonczar ?

**Comment by lgonczar** [comment]
Forgot about it. Will apply the comments.

**Comment by rkondratowicz** [comment]
Can we close it?

**Pull request closed without merging**